react/features/jaas/functions.js

```diff
@@ -1,9 +1,87 @@
 // @flow
 
-import { getVpaasTenant } from '../billing-counter/functions';
+import { jitsiLocalStorage } from '@jitsi/js-utils';
+import uuid from 'uuid';
 
+import { BILLING_ID, VPAAS_TENANT_PREFIX } from './constants';
 import logger from './logger';
 
+/**
+ * Returns the full vpaas tenant if available, given a path.
+ *
+ * @param {string} path - The meeting url path.
+ * @returns {string}
+ */
+export function extractVpaasTenantFromPath(path: string) {
```

Comment:
looks like wherever this is used externally, it does exactly what `getVpaasTenant` is doing. I would remove the export and replace the usages to `getVpaasTenant`